apps/studio/tests/lib/db.ts

```diff
@@ -770,15 +775,38 @@ export class DBTestUtil {
     expect(insertQuery).toBe(expectedQueries[this.dbType])
   }
 
+  async buildCreatePrimaryKeysAndAutoIncrementTests() {
+    const generator = new SqlGenerator(this.dialect, {
+      dbConfig: this.connection.server.config,
+      dbName: this.connection.database.database,
+    })
+    const schema: Schema = {
+      name: 'test_table',
+      columns: [{
+        columnName: 'id',
+        dataType: 'autoincrement',
+        primaryKey: true,
+        nullable: false,
+      }],
+    }
+    const query = generator.buildSql(schema)
+    const expectedQueries = {
```

Comment:
I think if we're expecting something different per-dialect, these should just go directly in the driver tests, but it's fine for now